packages/react-querybuilder/src/utils/formatQuery/formatQuery.SQL.test.ts

```diff
@@ -566,3 +568,15 @@ describe('non-standard combinators', () => {
     expect(formatQuery(queryForXor, 'sql')).toBe(`(f1 = 'v1' xor f2 = 'v2')`);
   });
 });
+
+it('Different quote character', () => {
+  expect(formatQuery(query, {
+    format: 'sql',
+    quoteChar: `'`,
+  })).toBe(sqlString);
+
+    expect(formatQuery(query, {
```

Comment:
The inconsistent indentation here indicates you need to run prettier. You can run it on the entire codebase with `bun pretty-print`.